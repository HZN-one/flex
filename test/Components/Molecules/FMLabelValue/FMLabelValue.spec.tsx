/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";

import { FMLabelValue } from "@Molecules";

import { render } from "@testing-library/react";

describe("FMLabelValue", () => {
  it("should match snapshot", () => {
    const { container } = render(<FMLabelValue label="Title" value="value" />);
    expect(container).toMatchSnapshot();
  });
});
