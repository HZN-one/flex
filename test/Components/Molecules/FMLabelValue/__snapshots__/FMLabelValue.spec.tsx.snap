// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMLabelValue should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-1qm1lh"
    data-testid="label-value-test"
  >
    <p
      class="MuiTypography-root MuiTypography-captionSemiBold MuiTypography-paragraph css-1uzr7kb-MuiTypography-root"
      data-testid="typography-label"
    >
      Title
    </p>
    <p
      class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
      data-testid="typography-test"
    >
      value
    </p>
  </div>
</div>
`;
